examples/step-58/doc/results.dox

```diff
@@ -0,0 +1,207 @@
+<h1>Results</h1>
+
+Running the code results in screen output like the following:
+```
+Number of active cells: 4096
+Number of degrees of freedom: 16641
+
+Time step 1 at t=0
+Time step 2 at t=0.00390625
+Time step 3 at t=0.0078125
+Time step 4 at t=0.0117188
+[...]
+```
+Running the program also yields a good number of output files that we will
+visualize in the following.
+
+
+<h3>Visualizing the solution</h3>
+
+The `output_results()` function of this program generates output files that
+consist of a number of variables: The solution (split into its real and imaginary
+parts), the magnitude, and the phase. If we visualize these four fields, we get
+images like the following after a few time steps (at time $t=0.242$, to be
+precise:
+
+<div class="twocolumn" style="width: 80%">
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.re.png"
+         alt="Real part of the solution at t=0.242"
+         width="400">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.im.png"
+         alt="Imaginary part of the solution at t=0.242"
+         width="400">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.magnitude.png"
+         alt="Magnitude of the solution at t=0.242"
+         width="400">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.phase.png"
+         alt="Phase of the solution at t=0.242"
+         width="400">
+  </div>
+</div>
+
+While the real and imaginary parts of the solution shown above are not
+particularly interesting (because, from a physical perspective, the
+global offset of the phase and therefore the balance between real and
+imaginary components, is meaningless), it is much more interesting to
+visualize the magnitude $|\psi(\mathbf x,t)|$ and phase
```

Comment:
Wouldn't it make sense to plot  the square of the amplitude $|\psi(\mathbf x,t)|^2$ instead of the magnitude. The square of the amplitude is the probability distribution of finding a particle.

Reply:
Yes, that's a fair point. I guess I have to redo all of my figures again :-(

Reply:
It is not big deal if you plot the amplitude instead of the square of the amplitude :)